package.json

```diff
@@ -8,6 +8,7 @@
   "license": "AGPL-3.0",
   "scripts": {
     "bootstrap": "lerna bootstrap",
+    "benchmark:browser": "lerna run benchmark --scope xod-client-browser --stream",
```

Comment:
It will be nice to add benchmark script into CI pipeline and store benchmarks somewhere (GCS?) to detect speed changes. 
Or just fail the test if the benchmark runs more than allowed time. In this case, this time will have to increase in the same PR, and that will pay attention to an optimization problem or will have to optimize the just added code.
We should discuss that, I think.
Maybe it should be implemented later in other PR.

Reply:
Yep, the original plan was to store profiles in GCS and implement it in another PR.